tests/EventListener/ResizeFormListenerTest.php

```diff
@@ -25,6 +25,9 @@
  */
 class ResizeFormListenerTest extends TestCase
 {
+    /**
+     * @group legacy
+     */
```

Comment:
You should use a class-level annotation instead of marking every test as part of this group ;)